device/rsvp/rsvpInterface.yaml

```diff
@@ -0,0 +1,81 @@
+components:
+  schemas:
+    Rsvp.Interface:
+      x-status: under-review
+      description: >-
+        Configuration for RSVP Interface.
+      type: object
+      required: [ipv4_name,neighbor_ip]
+      properties:
+        ipv4_name: 
+          description: >-
+            The globally unique name of the IPv4 interface connected to the DUT.            
+          type: string
+          x-constraint:
+          - '/components/schemas/Device.Ipv4/properties/name'
+        neighbor_ip:
+          description: >-
+            IPv4 address of the RSVP neighbor on this interface.
+          type: string
+          format: ipv4
+        label_space_start:
+          description: >-
+            The user-defined label space start value. The LSPs for which this router acts as a egress are assigned labels from this
+            label pool. 
+          type: integer
+          default: 1000
+          minimum: 0
+          maximum: 1048575
+        label_space_end:
+          description: >-
+            The user-defined label space end value.
+          type: integer
+          default: 100000
+          minimum: 0
+          maximum: 1048575
+        enable_refresh_reduction:
+          description: >-
+            Enables sending of Refresh Reduction as described in RFC2961.
+          type: boolean
+          default: false
+        summary_refresh_interval:
+          description: >-
+            The number of seconds between transmissions of successive Summary Refreshes.
```

Comment:
A description like "The number of seconds between transmissions of successive Summary Refreshes."  does not sound good. Once I got a tip from the Tech pub team that user-facing documentation should be written in the second person.  So above description can be written as "The number of seconds between transmissions of successive summary refreshes that **you** want to configure."
I am keeping this comment as place holder here. Since there are many instances of such descriptions.

Reply:
For now not modifying to second person ( your...) . This is because in vendor documentation or openconfig/yang model documentation, we could not find any such usage. Also , the modeling guide for writing descriptions for OTG model does not recommend this style: https://github.com/open-traffic-generator/models/blob/master/MODEL-GUIDE.md .